foreach example in document is incorrect

given the following main.go:

```go
package main

import (
	"fmt"
	"log"

	"github.com/minio/simdjson-go"
)

func main() {
	// Parse JSON:
	pj, err := simdjson.Parse([]byte(`{"Image":{"URL":"http://example.com/example.gif"}}`), nil)
	if err != nil {
		log.Fatal(err)
	}

	// Iterate each top level element.
	_ = pj.ForEach(func(i simdjson.Iter) error {
		fmt.Println("Got iterator for type:", i.Type())
		element, err := i.FindElement(element, "Image", "URL")
		if err == nil {
			value, _ := element.Iter.StringCvt()
			fmt.Println("Found element:", element.Name, "Type:", element.Type, "Value:", value)
		}
		return nil
	})

	// Output:
	// Got iterator for type: object
	// Found element: URL Type: string Value: http://example.com/example.gif
}
```
The following error is produced:

```
./main.go:20:33: undefined: element
```

Presumably, the user needs to initialize a (reusable?) `*simdjson.Element`, but this is not mentioned and the [documentation for `*Iter.FindElement`](https://pkg.go.dev/github.com/minio/simdjson-go#Iter.FindElement) does not provide any guidance on `*simdjson.Element` reuse. Could this be clarified? Is it safe to call `*iter.FindElement` multiple times to manually extract values into a struct? Further if `WithCopyStrings(false)` is set, is it okay to use these values, provided they are copied?

--- a/README.md
+++ b/README.md
@@ -59,7 +59,7 @@ func main() {
 	// Iterate each top level element.
 	_ = pj.ForEach(func(i Iter) error {
 		fmt.Println("Got iterator for type:", i.Type())
-		element, err := i.FindElement(element, "Image", "URL")
+		element, err := i.FindElement(nil, "Image", "URL")
 		if err == nil {
 			value, _ := element.Iter.StringCvt()
 			fmt.Println("Found element:", element.Name, "Type:", element.Type, "Value:", value)

